modules/ROOT/pages/configure-cidr-range.adoc

```diff
@@ -0,0 +1,24 @@
+= Configure Pod CIDR Range for Runtime Fabric
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+To configure the pod Classless Inter-Domain Routing (CIDR) range on an existing Runtime Fabric cluster:
+
+. Create a resource definition using a new `podCIDR` value. In the following example, the yaml file is named `pod-cidr.yaml` and `x.x.x.x/16` represents the new pod CIDR range:
```

Comment:
```suggestion
. Create a resource definition using a new `podCIDR` value. 
In the following example, the YAML file is named `pod-cidr.yaml` and `x.x.x.x/16` represents the new pod CIDR range:
```

Reply:
I was thrown off a bit by the mention of the YAML file name in the following example, because that does not appear until the following step example, and not the one directly below.